kuma/wiki/migrations/0005_auto_20190805_0306.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.22 on 2019-08-05 03:06
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('wiki', '0004_auto_20190708_1314'),
```

Comment:
```suggestion
        ('wiki', '0008_auto_20190914_2209'),
```
because new migrations have been added since.